### PR TITLE
small spelling mistake

### PR DESCRIPTION
**Pull request by tristansokol**
logits -> digit

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-examples/113)

### CODE REVIEW

**Comment by nsthorat** [comment]
`y` actually in this case should be the logits or the label vector. Just curious -- why do you think this is a mistake?

**Comment by tristansokol** [comment]
Oh sorry. I had never heard of the word `logits` and didn't encounter it anywhere else in the source code or in the html example: https://js.tensorflow.org/tutorials/mnist.html

Do you have a good definition for what `logits` means? The [wiki](https://en.wikipedia.org/wiki/Logit) definition doesn't really seem transposable with `label vector`

**Comment by nsthorat** [comment]
Check out the definition here: https://developers.google.com/machine-learning/glossary/#logits

Logits typically are the unnormalized predictions, right before the softmax normalization function.
